src/components/AddProfileForm.js

```diff
@@ -0,0 +1,37 @@
+import "./AddProfileForm.css";
+
+export default function AddProfileForm() {
+  return (
+    <div className="Form-Wrapper">
```

Comment:
To follow the same convention always, i'd call the class of  the root element of each component with the name of the component.
That will also imply some changes in your CSS :)

Reply:
I will change it then to AddProfileForm 😅